tests/marqo_test.py

```diff
@@ -52,19 +52,26 @@
 
 ## NOTES ABOUT THE TEST SUITE BEHAVIOUR ##
 
-When `cluster.is_marqo_cloud` is set to True, the tests will have different setUp and tearDown procedures:
-- Indices will not be deleted after each test.
-- However, documents will be deleted on every call for index.
+The index creation and deletion is handled differently for cloud and open-source tests.
 
+
+1) Cloud Environment:
+When `cluster.is_marqo_cloud` is set to True:
+- All the indexes will be created at the start of the test suite, in file
```

Comment:
nit: I prefer `test run` over `test suite` as "suite" is more about the code and "run" evokes a program in flight